Packs/CortexXDR/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+## [Unreleased]
```

Comment:
currently this file is automatically generated for new content entities (in the clear release notes process).

will it be included in the packs from day one or from the first modification?